wagtail/models/sites.py

```diff
@@ -16,33 +17,54 @@
 MATCH_HOSTNAME = 3
 
 
+class ClearableLocal(Local):
```

Comment:
Suggestion: This should live elsewhere, perhaps in `utils`

Reply:
Yeah, I like that :)

Reply:
Resolved in [a50fcb9](https://github.com/wagtail/wagtail/pull/10018/commits/a50fcb9fbf5ce3077f1c5013d71117dedaa0a5c1)